src/nasui/src/setupTests.ts

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
Is this useful? It seems that there is no main logic code.

Reply:
It's auto-generated. Removing it for now.